deployment/live/gcp/cloudbuild/prod/terragrunt.hcl

```diff
@@ -0,0 +1,18 @@
+terraform {
+  source = "${get_repo_root()}/deployment/modules/gcp//cloudbuild"
+}
```

Comment:
can this be moved to the ../terragrunt.hcl file?

Reply:
Moved.